docs/docsite/rst/playbooks_async.rst

```diff
@@ -33,9 +35,9 @@ poll value is 10 seconds if you do not specify a value for `poll`::
    default.
 
 Alternatively, if you do not need to wait on the task to complete, you may
-"fire and forget" by specifying a poll value of 0::
+"fire and forget" by specifying a poll value of 0:
```

Comment:
Please don't use this phrase in the docs. Please change all occurrences of the phrase "fire and forget" to "asynchronously launch a task".